test/suite/test_cursor_bound08.py

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/env python
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+import wiredtiger, wttest
+from wtscenario import make_scenarios
+from wtbound import bound_base
+from wiredtiger import stat
+
+# test_cursor_bound08.py
+# Test that the statistics added for cursor bound API are appropiately incremented for all different cursor
+# operations and edge cases.
+class test_cursor_bound08(bound_base):
+    conn_config = 'statistics=(all)'
+    file_name = 'test_cursor_bound08'
+    value_format = 'S'
+    lower_inclusive = True
+    upper_inclusive = True
+
+    types = [
+        ('file', dict(uri='file:', use_colgroup=False)),
+        ('table', dict(uri='table:', use_colgroup=False)),
+    ]
+
+    key_format_values = [
+        ('string', dict(key_format='S')),
+        # FIXME-WT-9489: Uncomment once column store search near is implemented.
+        # ('var', dict(key_format='r')),
+        ('int', dict(key_format='i')),
+        ('bytes', dict(key_format='u')),
+        ('composite_string', dict(key_format='SSS')),
+        ('composite_int_string', dict(key_format='iS')),
+        ('composite_complex', dict(key_format='iSru')),
+    ]
+
+    evict = [
+        ('inclusive-evict', dict(evict=True)),
+        ('inclusive-no-evict', dict(evict=False)),
+    ]
+
+    scenarios = make_scenarios(types, key_format_values, evict)
+
+    def create_session_and_cursor_timestamp(self):
```

Comment:
I take it we couldn't merge this with the wtbound class as it required timestamps?

Reply:
Yep, that's what happened.